crates/walrus-service/src/node/blob_event_processor.rs

```diff
@@ -259,6 +333,7 @@ impl BlobEventProcessor {
             background_processor_senders: senders,
             _background_processors: workers,
             sequential_processor,
+            background_per_processor_pending_event_count,
```

Comment:
Question: Just to be sure: If we have a sequential processor, we don't need to keep track of the pending event counts?

Reply:
Correct, because the pending event count will always be 0 since the observer, BlobEventProcessor::process_event will always process them online.